src/workers/userActivity.worker/inactivityTracker/queries/queries.js

```diff
@@ -1,8 +1,14 @@
-const getLastAppOpenedTimestamp = `query getUsers{
-  user_actions{
-    user_id,
-    last_app_opened_timestamp
+const getLastAppOpenedTimestamp = `
+  query getUsers($offset: Int, $limit: Int) {
+    user_actions(
+      order_by: { last_app_opened_timestamp: asc_nulls_last }
```

Comment:
Instead of nulls last, directly filter out the ones with null, using where block